.travis.yml

```diff
@@ -25,17 +24,16 @@ script:
   - git fetch origin +:refs/remotes/origin/HEAD; if ! git diff --quiet origin/HEAD HEAD -- deps/build.jl; then unset JULIA_TZ_VERSION; fi
   - |
     julia -e '
-      VERSION >= v"0.7.0-DEV.3656" && using Pkg
```

Comment:
I think you can ditch all of the custom logic here now and just use the Travis default steps.

Reply:
I may do that as another PR as https://docs.travis-ci.com/user/languages/julia/#coverage has sent me down a rabbit hole of changes.

Reply:
Addressed here: https://github.com/JuliaTime/TimeZones.jl/pull/170